clippy_lints/src/non_expressive_names.rs

```diff
@@ -69,7 +69,7 @@ struct SimilarNamesLocalVisitor<'a, 'tcx: 'a> {
 // this list contains lists of names that are allowed to be similar
 // the assumption is that no name is ever contained in multiple lists.
 #[cfg_attr(rustfmt, rustfmt_skip)]
-const WHITELIST: &'static [&'static [&'static str]] = &[
+const WHITELIST: &[&[&str]] = &[
```

Comment:
C'mon, It's my first time implementing a lint, even touching something relating to compiler stuff, please be tolerant. Even if it is disabled by default I would be happy :smile: